client/state/user-settings/selectors.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns whether there's currently a password change pending
+ *
+ * @param {object} state Global state tree
+ */
+export const isPendingPasswordChange = ( state ) => {
```

Comment:
There's a bunch of other selectors which I will likely move to this place as well over the course of the removal of `lib/user-settings`.